cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/poller/context/NetworkInterfaceDeleteCheckerContext.java

```diff
@@ -0,0 +1,24 @@
+package com.sequenceiq.cloudbreak.cloud.arm.poller.context;
+
+import com.sequenceiq.cloudbreak.cloud.arm.view.ArmCredentialView;
+
+public class NetworkInterfaceDeleteCheckerContext extends ArmStatusCheckerContext {
+
+    private String groupName;
+    private String networkName;
+
+    public NetworkInterfaceDeleteCheckerContext(ArmCredentialView armCredentialView, String groupName, String networkName) {
```

Comment:
Instead of NetworkInterfaceDeleteCheckerContex, maybe the NetworkInterfaceCheckerContext would be sufficient. And also in case of other Context , maybe the 'Delete' word could be eliminated